fedmsg.d/config.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import logging
+
+
+config = {
+    'ciconsumer': True,
```

Comment:
Is this necessary? What does it do?

Reply:
That was accidentally copied over from resultsdb-updater. I'll remove it.